### PR TITLE
Refactoring of the Why Wasabi section.

### PR DESCRIPTION
**Pull request by SimoneBovi**
I refactored the "Why Wasabi" section in order to make it consistent with the other ones.

Specifically, every other sections have a main page only containing the chapters, so I did the same here.

Then I moved the "Gaining Privacy with Wasabi Wallet for Bitcoin transactions" at the third place as I think it follows better the mental flow from Privacy in general -> Bitcoin - Wasabi.

### CODE REVIEW

File: docs/why-wasabi/GainingPrivacyWasabi.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Wasabi Wallet
+
+Wasabi Wallet is an *open source*, *non-custodial*, *privacy-focused*, desktop Bitcoin wallet offered by zkSNACKs Ltd.
+It differentiates itself from many other wallets for its strong *focus on user privacy*.
+
+Wasabi's goal is to *not only provide the traditional privacy* of our banks but to go even further and *guarantee privacy against Wasabi itself*.
+So not even the service provider (zkSNACKs) will be able to track your money.
+
+### The vision for a better Bitcoin
```
Comment (yahiheb) [inline docs/why-wasabi/GainingPrivacyWasabi.md]:
```suggestion
## The vision for a better Bitcoin
```

File: docs/why-wasabi/GainingPrivacyWasabi.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The motivation behind Wasabi is to provide the *strongest privacy for its users* that is technically possible today.
+
+### Technology behind Wasabi
```
Comment (yahiheb) [inline docs/why-wasabi/GainingPrivacyWasabi.md]:
```suggestion
## Technology behind Wasabi
```

**Comment by yahiheb** [comment]
Maybe we should put this as 1st chapter.

**Comment by SimoneBovi** [comment]
> Maybe we should put this as 1st chapter.

Yeah maybe, the issue is that in the Why Wasabi section there are considerations about privacy in general and then specifically regarding Bitcoin. This is why I put this as 3rd chapter, as you have to read and understand the first 2 chapters before to have a clue on why Wasabi could help. 

Also this follows the flow of the introductory paragraph that there is before the chapters list.

These were my thoughts, but I understand this could be debatable.

**Comment by yahiheb** [comment]
I understand that, but I still think this new chapter should be put first as there is no details in it, just a simple introduction of the privacy problem in Bitcoin and what Wasabi is ...
On the other hand the other two chapters contain more details about privacy.

**Comment by RiccardoMasutti** [comment]
We should wait to merge this.
Maybe after #341?

**Comment by SimoneBovi** [comment]
> We should wait to merge this.
> Maybe after #341?

I don't see any conflict.

**Comment by SimoneBovi** [comment]
I see what you mean and this is a good point, thanks for the review!

To me consistency is a really great feature that could help a reader.
I thought that it was the "Why Wasabi" section not consistent with the other ones and not viceversa, this is why I proposed this PR :)

I honestly do not think that more introduction is needed for the other pillars other than the already present first paragraphs that (maybe) could be slightly expanded.

I agree that there is some redundancy with the chapters also on the sidebar but I think it could be useful from a User Experience point of view.

Also, as it stands right now, the README for Why Wasabi contains other links other than the chapters, that is the ones in "TECHNOLOGY BEHIND WASABI". From a UX perspective I do not think it is a good thing to put this kind of inline links in an Introductory page, this could create confusion. And that was my rational behind putting everything as a new chapter.

Or at least that were my thoughts when first reading it :)

**Comment by MaxHillebrand** [comment]
Yes @SimoneBovi your PR is definitely justified, as we do need to change what is currently, it is an awkward work in progress right now, there is clearly no consistency.

> I agree that there is some redundancy with the chapters also on the sidebar but I think it could be useful from a User Experience point of view.

Yes, I click on these links all the time, as they are more compact than the sidebar menu - so I am not in favor of removing the links. But I'm saying that having a README with only the links might be a bit odd...

> I honestly do not think that more introduction is needed for the other pillars other than the already present first paragraphs that (maybe) could be slightly expanded.

Yes, these READMEs shouldn't have a wealth of additional knowledge, they should be a short and precise introduction to entice the reader for further inquiry. I'm not entirely certain, but I think it's a good idea to have infographics, pictures and videos in these sections, which are pleasant for a quick skim of the topic.

On second thought, this lower section in `/why-wasabi/README.md` would make for a good standalone chapter... 

I'm not so sure, let me think about it for a bit, and the @ others please also give your thoughts on what to do.

**Comment by yahiheb** [comment]
I think we should keep things as they are, and proceed with merging this PR.
I don't think we need more introductions, maybe the existing ones should be improved a bit but that is all.

**Comment by MaxHillebrand** [comment]
OK, then we do.

Now one more thing is, that I believe it would be better to have this chapter as 3rd, so that it is `Why Privacy` `Privacy in Bitcoin` and `Privacy with Wasabi`. This makes more sense from a story telling point of view.

**Comment by RiccardoMasutti** [comment]
I'm still not convinced of this restructuring but go for it.

**Comment by SimoneBovi** [comment]
> OK, then we do.
> 
> Now one more thing is, that I believe it would be better to have this chapter as 3rd, so that it is `Why Privacy` `Privacy in Bitcoin` and `Privacy with Wasabi`. This makes more sense from a story telling point of view.

That was my first idea!

Last question before I push new changes: the ones that you have written are the exact titles of the chapters that you want or they are just shortened?

For the sake of clarity these are the titles until my last commit, in this specific order:

- Gaining Privacy with Wasabi Wallet for Bitcoin transactions

- Why Privacy is Important

- Privacy in Bitcoin

- Ten Commandments

- Transaction Surveillance Companies

Is it ok for you if I just move the 1st chapter to the 3rd place leaving these titles as they are or do you want also to change them?

**Comment by MaxHillebrand** [comment]
The titles are good as they are now, I was talking about the ordering

My preferred order is 

-    Why Privacy is Important
-    Privacy in Bitcoin
-    Gaining Privacy with Wasabi Wallet for Bitcoin transactions
-    Ten Commandments
-    Transaction Surveillance Companies

@yahiheb you were previously not in favor if this, what do you say?

**Comment by yahiheb** [comment]
I agree with you guys, it is better for this chapter to be 3rd.

**Comment by SimoneBovi** [comment]
Moved to the 3rd place.

**Pull request merged**